frotz.py

```diff
@@ -143,19 +178,19 @@ def read_file(self, file_path):
 
     def open_url(self, url):
         """Ask the journal to open an URL for us."""
-        from sugar import profile
+        from sugar3 import profile
         from shutil import rmtree
-        from sugar.datastore import datastore
-        from sugar.activity.activity import show_object_in_journal
+        from sugar3.datastore import datastore
+        from sugar3.activity.activity import show_object_in_journal
         from tempfile import mkdtemp
         tmpfolder = mkdtemp('.tmp', 'url', os.path.join(self.get_activity_root(), 'instance'))
         tmpfilepath = os.path.join(tmpfolder, 'url')
         try:
             tmpfile = open(tmpfilepath, 'w')
             tmpfile.write(url)
             tmpfile.close()
-            os.chmod(tmpfolder, 0755)
-            os.chmod(tmpfilepath, 0755)
+            os.chmod(tmpfolder, 0o755)
+            os.chmod(tmpfilepath, 0o644)
```

Comment:
https://github.com/sugarlabs/Frotz/pull/7/commits/607dd6ff8a5704fbe14a960443de12e6bddf9721 doesn't explain in the commit message _why_ you are changing the file mask.

Reply:
https://github.com/sugarlabs/Frotz/pull/7/commits/69f178133e99c6ec2001a187df50a027eb8e997a Please see